### PR TITLE
Flowai tool

### PR DESCRIPTION
**Pull request by emily101-gif**

### CODE REVIEW

File: flowtools/flowai/FCSflowAI.xml
Comment (pcm32) [inline flowtools/flowai/FCSflowAI.xml:4]: My only concern is whether we should stick to a specific version of flowcore or go with whatever flowcore version flowai wants to use. Do you know anyone in the field that normally uses these packages that could advice @emily101-gif ? My question is whether it might be relevant to pin flowcore across all modules to the same version or not.

**Review by pcm32 — COMMENTED** [review]
Besides doubt about pining or not flowcore, I think that this is good to go.

File: flowtools/flowai/FCSflowAI.xml
Comment (emily101-gif) [reply flowtools/flowai/FCSflowAI.xml:4]: Hi Pablo - I'm afraid I don't know any experts other than just contacting bioconductor. But for many of the tools when I tried to use alternative versions of flowCore than in the original xml file fails. Is it flowCore 1.42.0 we want to be sticking with then, and what is the reason we can't use different versions of packages?

Comment (pcm32) [reply flowtools/flowai/FCSflowAI.xml:4]: which version did you try? What I mean here is not to change the version of flowcore manually, but to just use flowai dependency on its own and let conda decide which version of flowcore to install (since flowai depends on flowcore in conda).

Comment (pcm32) [reply flowtools/flowai/FCSflowAI.xml:4]: Based on the comment of the maintainers from flowcore, I'm going to stick to a single dependency. Will merge like this if tests pass.

**Comment by pcm32** [comment]
There is currently an issue with the bioconda package for flowai, is probably broken. Have reported and we'll see.

**Comment by emily101-gif** [comment]
> There is currently an issue with the bioconda package for flowai, is probably broken. Have reported and we'll see.

Ok thanks Pablo.

**Review by pcm32 — APPROVED** [review]

**Pull request merged**
